parl/remote/job.py

```diff
@@ -60,12 +60,12 @@ def __init__(self, worker_address):
         self.worker_address = worker_address
         self.job_ip = get_ip_address()
         self.pid = os.getpid()
-        self.lock = threading.Lock()
```

Comment:
Why do you move this line down?

Reply:
In Windows, it will raise errors when creating `threading.Lock` before starting `multiprocess.Process` in init function of a class.

I have added comments about it.